### PR TITLE
Also able to use backrest init <name> to create a folder, <name>

### PR DESCRIPTION
**Pull request by Beeze**

### CODE REVIEW

**Comment by Beeze** [comment]
@passabilities

File: bin/backrest-init.js [hunk 1]
```diff
@@ -11,11 +11,13 @@ program
       name
     }
 
+    let projectPath = `${process.cwd()}/${name}`
+    fs.mkdirSync(projectPath)
```
Comment (Beeze) [inline bin/backrest-init.js]: This line may seem redundant, but if the folder isn't already made prior to us going into the `_.each`, it throws an error when creating the `/app` folder

File: bin/backrest-init.js [hunk 2]
```diff
@@ -11,11 +11,13 @@ program
...
     _.each(glob.sync(`${__dirname}/../template/**/*`), (file) => {
-      let pathName = `${process.cwd()}/${_.last(file.split('template'))}`
+      let pathName = `${projectPath}/${_.last(file.split('template/'))}`
       if(fs.statSync(file).isDirectory()) {
-        console.log(`Creating directory '${path.dirname(file)}'`)
-
+        console.log(`Creating directory '${path.dirname(pathName)}'`)
         fs.mkdirSync(pathName)
       } else {
         console.log(`Building file ${path.basename(file)}`)
```
Comment (passabilities) [inline bin/backrest-init.js]: i believe `file` needs to be changed to `pathName` here as well

File: bin/backrest-init.js [hunk 3]
```diff
@@ -11,11 +11,13 @@ program
...
+    let projectPath = `${process.cwd()}/${name}`
+    fs.mkdirSync(projectPath)
+
     _.each(glob.sync(`${__dirname}/../template/**/*`), (file) => {
-      let pathName = `${process.cwd()}/${_.last(file.split('template'))}`
+      let pathName = `${projectPath}/${_.last(file.split('template/'))}`
       if(fs.statSync(file).isDirectory()) {
-        console.log(`Creating directory '${path.dirname(file)}'`)
-
+        console.log(`Creating directory '${path.dirname(pathName)}'`)
```
Comment (passabilities) [inline bin/backrest-init.js]: lets add that new line back in there to keep it clean and consistent :)

**Comment by passabilities** [comment]
👍

**Pull request merged**